### PR TITLE
Random Battle: Improve Archeops'  sets

### PR DESCRIPTION
**Pull request by ascriptmaster**
Archeops gets Acrobatics as a more reliable Flying STAB.

Landorus gets Rock Slide as a Sheer Force-boosted Rock move for coverage. (Now split to #2604)

-----

@TheImmortal 

Acrobatics has > 50% usage on Archeops. Even if it denies it the strength of Banded Head Smash, there isn't really any reason Pluck should be on its list.

Rock Slide has around 30% usage on Landorus. Coverage is nice, especially with the randbats flat-EV policy and it coming off of Lando's stronger 125 base Attack.

### CODE REVIEW

**Comment by ascriptmaster** [comment]
Speaking of Archeops, actually, why the heck does it get THREE Rock STABs in its pool?

**Comment by ascriptmaster** [comment]
I'm going to split the PR. Lando will go in a different PR because it's going to be less controversial afaik

**Comment by ascriptmaster** [comment]
@TheImmortal any comments on these changes?

**Comment by TheImmortal** [comment]
Perhaps reject Acro and not Pluck to increase the likelihood of Choice sets.

**Comment by ascriptmaster** [comment]
Does Archeops really need the Band that badly? An NU pokemon running off 140 base Attack / 110 base Speed can still be quite threatening, and also Pluck has really low base power. (Also Acrobatics is really spammable and is still a really solid attack)

**Comment by ascriptmaster** [comment]
@TheImmortal please reply to my query or I can't make the change

**Comment by TheImmortal** [comment]
I did some stuff.

Acrobatics gives it Flying Gem.
Removed Pluck and gave it Aqua Tail instead. I believe it hits more things and has the same BP. Needed as the 4th move on a Choice Band set.
Added Tailwind to give it the chance to get Expert Belt, when it doesn't get Acrobatics. Tailwind is a good move for it to use once Defeatist activates, before it dies.

Also removed the Flying types get Flying STAB check thing because a lot of Flying types don't run Flying STAB. Dragonite, Altaria, etc. And I needed to removed that so it doesn't always get Acrobatics and has the chance of getting Choice Band.

tl;dr: Flying Gem will be its most common set but it has a chance of Choice Band and Expert Belt too.
